### DISCUSSION TITLE
Use a Streamdeck to control MIDI devices?

### DISCUSSION THREAD

**Discussion by dhtseany** [streamdeck-linux-gui/streamdeck-linux-gui#172] [Q&A]
Hi, I'm a Linux user (Arch) I want to control certain aspects of my sources in jack_mixer and according to [their man page info](https://man.archlinux.org/man/extra/jack_mixer/jack_mixer.1.en) I can manipulate various aspects of jack_mixer by sending MIDI commands which specify the changes I want to make, such as muting a channel or increasing/decreasing the fader levels. 

The problem is that I can't seem to find any how-to-guides that cover this topic beyond the man page I linked and I'd like use my Streamdeck to send those MIDI control commands. Has anyone successfully sent MIDI commands from a Streamdeck via streamdeck-ui to control jack_mixer before? Thanks for any help in advance!

**Comment by dhtseany** [comment]
Last night I continued my research after posting this thread and I came across [jack_mixer_cc](https://github.com/sen87/jack_mixer_cc), which should take care of my needs for the time being however I was still sort of hoping to have a better understanding of how other streamdeck-ui users integrate MIDI control with their decks.

**Comment by Da-Boom** [comment]
I use a sendMIDI command - https://github.com/gbevin/SendMIDI (Its also on the AUR)
Works quite well
